perfmetrics/scripts/README.md

```diff
@@ -0,0 +1,82 @@
+# GCSFuse Metrics Extraction
+This project will be used to run FIO load tests on a GCP VM and extract the GCSFuse metrics from these load tests into a Google Sheet. This can be used to monitor the performance of GCSFuse and make appropriate improvements in future.
+
+## Installing required packages
+### FIO
+```bash
+sudo apt-get update
+sudo apt-get install fio -y
+```
+### GCSFuse
+```bash
+GCSFUSE_VERSION=0.41.1
+curl -L -O https://github.com/GoogleCloudPlatform/gcsfuse/releases/download/v$GCSFUSE_VERSION/gcsfuse_"$GCSFUSE_VERSION"_amd64.deb
+sudo dpkg --install gcsfuse_"$GCSFUSE_VERSION"_amd64.deb
+```
+
+## How to run
+1. Create a GCP VM with OS version as Ubuntu 20.04. Follow this [documentation](https://cloud.google.com/compute/docs/create-linux-vm-instance) and start your VM. Follow the next steps in your VM.
+2. Install the required packages as mentioned in the above section
+3. Clone the GCSFuse repo and cd into the perfmetrics/scripts directory:
+```bash
+git clone https://github.com/GoogleCloudPlatform/gcsfuse.git
+cd gcsfuse/perfmetrics/scripts
+```
+4. Create a directory and mount your GCS bucket into it using GCSFuse:
+```bash
+mkdir -p your-directory-name
+GCSFUSE_FLAGS="--implicit-dirs --max-conns-per-host 100 --disable-http2"
+BUCKET_NAME=your-bucket-name
+MOUNT_POINT=your-directory-name
```

Comment:
Move these two lines ahead of the mkdir -p call

Reply:
done